src/custom/state/orders/actions.ts

```diff
@@ -21,6 +21,11 @@ export enum OrderStatus {
   REFUNDED = 'refunded',
 }
 
+export enum OrderClass {
```

Comment:
https://github.com/cowprotocol/cow-sdk/commit/0a99544ae5d9d4ff39e807a3ffdf416f14b455e1#diff-7bb820318d1e4352d1cd71225f5a30101b8b30cf1186af9b405eecc654e3ee68R13

I have updated the SDK, so we can import the type from it

Reply:
[v1.0.1-RC.2](https://github.com/cowprotocol/cow-sdk/releases/tag/v1.0.1-RC.2)

Reply:
If I update the cow-sdk, then we get into the issues of supported chains, and I think we should do that in a separate PR and leave this one as is.